modules/core/src/main/scala/sangria/parser/package.scala

```diff
@@ -0,0 +1,7 @@
+package sangria
+
+package object parser {
+
+  @deprecated("please use `sangria.ast.SourceMapper`", "3.0.0")
+  type SourceMapper = sangria.ast.SourceMapper
```

Comment:
Should we add a small test to make sure we provide this backwards compatibility?
During a migration from 2.x to 3.x, this is very important to avoid compilation errors.

Reply:
What are you thinking? What would the test be?

Reply:
Very simple, just using `sangria.parser.SourceMapper`.
```
val mapper: sangria.parser.SourceMapper = new DefaultSourceMapper...
succeeded
```

Reply:
Oh, like:
```scala
doesThisCompile {
  val mapper: sangria.parser.SourceMapper = new DefaultSourceMapper...
}
```
I haven't built a test like that yet.

I'm busy most of today, but can maybe get to it later.